_run/single/kustomize/akash-provider/kustomization.yaml

```diff
@@ -22,7 +22,7 @@ configMapGenerator:
     behavior: merge
     literals:
       - ingress-domain=app.localhost
-      - cluster-public-hostname=app.localhost
+      - cluster_public_hostname=app.localhost
```

Comment:
I don't really understand the motivation for this change, but if we're going change this argument then all of the remaining ones as well.

@boz What are we trying to accomplish by changing dashes to underscores?

Reply:
i think it relates to issue with `jq`.  in it's queries `-` has to be escaped or quoted

Reply:
I prefer `-`, but with `jq` it makes things more cumbersome.  You have to do something special to handle it.

```sh
jq '.["foo-bar"] | ."baz-boo"'
```
vs

```sh
jq '.foo_bar | .baz_boo'
```

It's also inconsistent with the style that is in tendermint and cosmos-sdk, maybe for this reason.

Reply:
I guess one thing I don't understand is why the command line arguments are being manipulated with `jq` ? 

But other than that, we'd need to change all of the flags for this to make sense.

Reply:
OH sorry.  Yes, these don't need to be changed.  Thank you.

@rsum1 : It's only the struct fields that have an annotation like `json:"foo-bar"` that need to be changed - this makes the field name be `"foo-bar"` in the JSON output/input.